app/src/app/components/contents/downloads/Downloads.tsx

```diff
@@ -0,0 +1,39 @@
+import { useParams } from "react-router-dom";
+import { usePacketOutletContext } from "../../main/PacketOutlet";
+import { PacketHeader } from "../packets";
+import { Card, CardContent } from "../../Base/Card";
+import FileRow from "./FileRow";
+import { OrderlyDownloads } from "./OrderlyDownloads";
+
+export default function Downloads() {
+  const { packetId, packetName } = useParams();
+  const { packet } = usePacketOutletContext();
+  const packetIsFromOrderly = !!packet?.custom?.orderly;
+
+  return (
+    <>
+      {packet && (
+        <>
+          <PacketHeader packetName={packetName ?? ""} packetId={packetId ?? ""} displayName={packet?.displayName} />
+          {packetIsFromOrderly && <OrderlyDownloads packet={packet} />}
+          {!packetIsFromOrderly && (
```

Comment:
We don't have any non-orderly packets in our demo set, do we? Maybe we should make a ticket to add one so we can easily manually test code like this.

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-6157/Add-non-orderly-packet-to-demos